docs/transformations.rst

```diff
@@ -23,6 +23,18 @@ Code example to make images inline:
     >>> message.html
     u'<html><body><img src="cid:promo.png"></body></html>'
 
+To inline CSS using `Premailer <https://github.com/peterbe/premailer>`_:
+
+.. code-block:: python
+
+    >>> html = '''<style>h1 { color: red }</style>
+    ...           <h1>Hello world!</h1>'''
+    >>> message = emails.Message(html=html)
+    >>> message.transformer.premailer.transform()
```

Comment:
Not sure. I don't like this code now. I think should add `message.transform` method that incapsulates `load_and_transform` & `save`